src/ui/atoms/inputBase/InputBase.tsx

```diff
@@ -95,7 +100,8 @@ export const InputBase = ({
     error: hasError
   })
   const textareaClass = classNames(inputClass, 'okp4-input-base-textarea', {
-    'disable-resize': multiline && disableAreaResize
+    'disable-resize': multiline && disableAreaResize,
+    'resize-height': fullWidth && multiline
```

Comment:
I will add a little bit more context to the class name because, by default, the text area can be resized vertically and horizontally.

```suggestion
    'resize-height-only': fullWidth && multiline
```